guacamole_iam.tf

```diff
@@ -1,6 +1,6 @@
 # Create a role that allows the instance to read its certs from S3.
 module "guacamole_certreadrole" {
-  source = "github.com/cisagov/cert-read-role-tf-module"
+  source = "github.com/cisagov/cert-read-role-tf-module?ref=test_tf_aws_provider_v3"
```

Comment:
Don't forget to revert this before merging.

Reply:
Reverted with https://github.com/cisagov/cool-assessment-terraform/pull/69/commits/e2e15c3783a5b48d0ac2dc3b2d1d3f1851d3cc3d.